cpp/src/reshape/explode.cu

```diff
@@ -118,13 +155,14 @@ std::unique_ptr<table> explode_functor::operator()<list_view>(
  *
  * @param stream CUDA stream used for device memory operations and kernel launches.
  */
+template <bool include_pos>
```

Comment:
Does this need to be a template parameter? From what I can tell, making it a function argument just adds a few extra conditional checks in host code, so the overhead would be trivial. Otherwise this doubles the binary size and compile time of this function.